src/EditModel/SharingNotification.component.js

```diff
@@ -45,4 +45,4 @@ const SharingNotification = React.createClass({
     },
 });
 
-export default SharingNotification;
+export default withAuth(SharingNotification);
```

Comment:
A HoC is much nicer. :) I'm glad someone finally got around to doing that.

Reply:
Very slick. Much easier to see where the added logic comes from.